src/patternfly/base/tokens/tokens-local.scss

```diff
@@ -146,7 +146,23 @@
     var(--pf-t--global--box-shadow--spread--lg)
     var(--pf-t--global--box-shadow--color--lg);
 
-  // Transitions
+  // Motion
+  .pf-m-no-motion {
+    // Set all motion tokens to 0 for testing purposes
```

Comment:
From the comment that it's for testing, this may not be relevant, but I think a good place for this would be [patternfly-common.scss](https://github.com/patternfly/patternfly/blob/493564c35a6768d4c23bd7f68bcd92e71bde66c2/src/patternfly/base/patternfly-common.scss). And just a thought, but if this should stomp on all animations, I wonder if we should consider adding `!important` to these declarations?

Reply:
I debated about where to put it. I think you're right. I'll move it and add the `!important`